source/_integrations/lawn_mower.mqtt.markdown

```diff
@@ -0,0 +1,222 @@
+      type: string
+dock_command_template:
+  description: Defines a [template](/docs/configuration/templating/#using-templates-with-the-mqtt-integration) to generate the payload to send to `dock_command_topic`. The `value` parameter in the template will be set to `dock`.
+  required: false
+  type: template
+dock_command_topic:
+  description: The MQTT topic that publishes commands when the service `lawn_mower.dock` service call is executed. The value `dock` is published when the service is called. Use a `dock_command_template` to publish a custom format.
+  required: false
+  type: string
+enabled_by_default:
+  description: Flag which defines if the entity should be enabled when first added.
+  required: false
+  type: boolean
+  default: true
+encoding:
+  description: The encoding of the payloads received and published messages. Set to `""` to disable decoding of the incoming payload.
+  required: false
+  type: string
+  default: "utf-8"
+entity_category:
+  description: The [category](https://developers.home-assistant.io/docs/core/entity#generic-properties) of the entity.
+  required: false
+  type: string
+  default: None
+icon:
+  description: "[Icon](/docs/configuration/customizing-devices/#icon) for the entity."
+  required: false
+  type: icon
+json_attributes_template:
+  description: "Defines a [template](/docs/configuration/templating/#using-templates-with-the-mqtt-integration) to extract the JSON dictionary from messages received on the `json_attributes_topic`."
+  required: false
+  type: template
+json_attributes_topic:
+  description: The MQTT topic subscribed to receive a JSON dictionary payload and then set as entity attributes. Implies `force_update` of the current activity state when a message is received on this topic.
+  required: false
+  type: string
+name:
+  description: The name of the lawn mower. Can be set to `null` if only the device name is relevant.
+  required: false
+  type: string
+object_id:
+  description: Used instead of `name` for automatic generation of `entity_id`
+  required: false
+  type: string
+optimistic:
+  description: Flag that defines if the lawn mower works in optimistic mode.
+  required: false
+  type: boolean
+  default: "`true` if no `state_topic` defined, else `false`."
+pause_command_template:
+  description: Defines a [template](/docs/configuration/templating/#using-templates-with-the-mqtt-integration) to generate the payload to send to `pause_command_topic`. The `value` parameter in the template will be set to `pause`.
+  required: false
+  type: template
+pause_command_topic:
+  description: The MQTT topic that publishes commands when the service `lawn_mower.pause` service call is executed. The value `pause` is published when the service is called. Use a `pause_command_template` to publish a custom format.
+  required: false
+  type: string
+qos:
+  description: The maximum QoS level of the state topic. Default is 0 and will also be used to publishing messages.
```

Comment:
I'm not sure I understand this. Is this saying that 0 is also used to publish messages?

Reply:
I'll have a look. I actually copy pasted that part.

Reply:
Found some more occurrences that need refrasing, will open a new PR targetting `current` for those once we agree on this one.